rpc/types_transaction.go

```diff
@@ -258,6 +258,17 @@ func (v *TransactionVersion) BigInt() (*big.Int, error) {
 	}
 }
 
+func (v *TransactionVersion) Felt() (*felt.Felt, error) {
+	switch *v {
+	case TransactionV0:
+		return &felt.Zero, nil
+	case TransactionV1:
+		return new(felt.Felt).SetUint64(1), nil
+	default:
```

Comment:
PR #294 adds `TransactionV2`, let's merge that one first and add it here as well, so this example can go in the rpcv03 release

Reply:
Other than this, I didn't see anything else in this pr that was dependent on rpcv03 changes. Will have to review again once the other prs are merged